corehq/apps/translations/templates/translations/base_template.html

```diff
@@ -12,18 +12,22 @@ <h2 class="text-hq-nav-header">{% trans "TOOLS" %}</h2>
         </li>
     </ul>
 
-    <h2 class="text-hq-nav-header">{% trans "TRANSIFEX" %}</h2>
-    <ul class="nav nav-hq-sidebar">
-        <li>
-            <a href="{% url 'icds_app_translations' domain %}">
-                {% trans "Manage App Translations" %}
-            </a>
-        </li>
-        <li>
-            <a href="{% url 'pull_resource' domain %}">
-                {% trans "Pull Resource" %}
-            </a>
-        </li>
-    </ul>
+    {% if transifex_details_available %}
+        {% if request|toggle_enabled:"APP_TRANSLATIONS_WITH_TRANSIFEX" %}
```

Comment:
It's more typical to use the toggle's `required_decorator` on the view, and then you also get a handy [message](https://github.com/dimagi/commcare-hq/pull/15485/) if the toggle isn't enabled.

Reply:
These is the base template which has links(one link for now, convert translations) which does not need this flag enabled. So its just to hide links that need the flag enabled. 
The views corresponding to the links that need the flag have the check at the view level.